### PR TITLE
Feature/store

### PR DESCRIPTION
**Pull request by JbirdL86**
What's new :question: 

- Add configureStore for redux store
- Add book reducer actions and actions creators
- Add functionality for adding books to the store
- Add functionality for removing books from the store
- Add redux-logger

### CODE REVIEW

**Review by wandji20 — CHANGES_REQUESTED** [review]
Hello, @JbirdL86  :raised_hands: 
-----
Great :muscle:  job :raised_hands:  implementing most of the requirements for this project.
However, there are a few issues you need to address to have this project approved.
## To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Very descriptive commit messages and pull request summary
:heavy_check_mark: Add and remove book functionalities :100: 


## Status: Require Changes :recycle: :red_circle: 
Please check the comments below and under the review and make all required changes.
Optional comments are not required to mark this project approved
- [ ] It will be nice to ignore all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, .hintrc, .eslintrc.json etc).
- [ ] It will be nice to update your README file and make it more customized to your project.


**Happy coding :computer:** 
**_Do not hesitate to make comments in this PR in case of doubts._**
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,70 +1,33 @@
...
+- Html, Javascript, CSS
+- Webpack
+- React JS
+- Redux
+- Developed in VSCode
+
+## Setup 
+### clone repository
+```bash
+$ git clone <git@github.com:JbirdL86/-app.git>
```
Comment (GzimAsani) [inline README.md]: - [ ] Add the correct link here

File: README.md [hunk 2]
```diff
@@ -1,70 +1,33 @@
...
+```
+### Start Project
+```bash
+$ cd react-todo-app
```
Comment (GzimAsani) [inline README.md]: - [ ] Add the correct name

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Required Changes :recycle: 

## Hello  @JbirdL86    hope you're well :dove: You have done great job so far :100:  

## To highlight:
- [ ] The gitflow has been followed :+1: 
- [ ] Readme almost perfect :hourglass: 

## To improve:
- [ ] Check inline comments for more information

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 You can also consider:

- [ ] Add a picture to your readme file, it'll look more professional :thought_balloon: 

 Cheers and Happy coding!

 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by GzimAsani — APPROVED** [review]
# Hello @JbirdL86         

## Thank you for implementing all the project requirements, yo've done a great job :trophy:  . Your project is **ACCEPTED :green_circle:** and I wish you the best of luck in your next projects :+1: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 You can also consider:


 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by JbirdL86** [comment]
@GzimAsani Thank you for your review. I'm just wondering what do you mean by "is not being appended to the page"?

**Comment by addod19** [comment]
### Hi @JbirdL86 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that I cannot review because the link to the project has already been approved and merged. Kindly find the right link and resubmit

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by dasileker — COMMENTED** [review]
### **Hi @JbirdL86,** 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, as a reply to your `PR` message you should not submit the old `PR` for review, instead create a new `branch` that contains the same codes and open a new `PR` then you are ready  to submit it for review



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

rejected


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._
`

**Pull request merged**